public/app/features/variables/custom/reducer.ts

```diff
@@ -36,9 +36,16 @@ export const customVariableSlice = createSlice({
       const instanceState = getInstanceState<CustomVariableModel>(state, action.payload.id);
       const { includeAll, query } = instanceState;
       const match = query.match(/(?:\\,|[^,])+/g) ?? [];
-
+      
       const options = match.map(text => {
         text = text.replace(/\\,/g, ',');
+        const textMatch = text.match(/(?:\\:|[^:])+/g) ?? [];
+        if (textMatch.length > 1) {
+          let [key, value] = textMatch;
+          key = key.replace(/\\:/g, ':');
+          value = value.replace(/\\:/g, ':');
```

Comment:
Have you added any test cases for this use case? I would either add tests for this or remove these lines.

Reply:
@hugohaggmark 
Do you think we need to cover for the case if someone has a semicolon within either the key or value? If not, i believe i can remove these lines.

Reply:
@sartaj10 I think you should keep them but then you should add a test for them. Also prettiers seems to be complaining on your branch so the build is failing :)

Reply:
Ok, i'll add that. Yeah, i need to figure out why that build is failing. Is there a way to check where that's happening before pushing the commit?

Reply:
I think prettier is complaining about the formatting in `public/app/features/variables/custom/reducer.ts`. Let me know if I could help out